phpmyfaq/admin/assets/src/content/news.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * News administration stuff
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * @package   phpMyFAQ
+ * @author    Thorsten Rinne <thorsten@phpmyfaq.de>
+ * @author    Jan Harms <modelrailroader@gmx-topmail.de>
+ * @copyright 2024 phpMyFAQ Team
+ * @license   https://www.mozilla.org/MPL/2.0/ Mozilla Public License Version 2.0
+ * @link      https://www.phpmyfaq.de
+ * @since     2024-04-20
+ */
+
+import { activateNews, addNews, deleteNews, updateNews } from '../api';
+import { Modal } from 'bootstrap';
+import { TinyMCE } from 'tinymce';
+import tinymce from 'tinymce/tinymce';
+
+export const handleAddNews = () => {
+  const submit = document.getElementById('submitAddNews');
+  if (submit) {
+    submit.addEventListener('click', async (event) => {
+      event.preventDefault();
+      let target = '';
+      document.querySelectorAll('#target').forEach(function (item) {

```

Comment:
You can use an arrow function here:

`document.querySelectorAll('#target').forEach(item) => {`